skimage/segmentation/active_contour_model.py

```diff
@@ -97,22 +94,6 @@ def active_contour(image, snake, alpha=0.01, beta=0.1,
     25
 
     """
-    if bc is not None:
-        message = ('The keyword argument `bc` to `active_contour` has been '
-                   'renamed. Use `boundary_condition=` instead. `bc` will be '
-                   'removed in scikit-image v0.18.')
-        warn(message, stacklevel=2)
-        boundary_condition = bc
-    if coordinates is None:
-        message = ('The coordinates used by `active_contour` will change '
-                   'from xy coordinates (transposed from image dimensions) to '
-                   'rc coordinates in scikit-image 0.18. Set '
-                   "`coordinates='rc'` to silence this warning. "
-                   "`coordinates='xy'` will restore the old behavior until "
-                   '0.18, but will stop working thereafter.')
-        warn(message, category=FutureWarning, stacklevel=2)
-        coordinates = 'xy'
-        snake_xy = snake
     if coordinates == 'rc':
         snake_xy = snake[:, ::-1]
```

Comment:
This if statement needs to be removed. My suggestion is that we leave `coordinates` as a do-nothing argument until version 1.0, at which point we can remove it. Potentially, we should raise a ValueError if coordinates is anything *other* than 'rc' now.

This kind of 4-version deprecation would be best to avoid — ie we don't want to tell people, use this argument to avoid warnings. Ok now stop using this argument altogether. So, I'm happy to leave coordinates around, but all values other than 'rc' should be an error.

Reply:
Thanks for the clarification. I'll do that.